src/main/java/com/iota/iri/storage/Tangle.java

```diff
@@ -107,16 +114,27 @@ public void delete(Class<?> model, Indexable index) throws Exception {
             return latest;
     }
 
-    public Boolean update(Persistable model, Indexable index, String item) throws Exception {
-            boolean success = false;
-            for(PersistenceProvider provider: this.persistenceProviders) {
-                if(success) {
-                    provider.update(model, index, item);
-                } else {
-                    success = provider.update(model, index, item);
-                }
-            }
-            return success;
+    public void update(Persistable model, Indexable index, String item) throws Exception {
```

Comment:
Although we never actually check the return of the update currently, im not sure if we should remove it. Lets wait for @GalRogozinski to come back from holiday and see what he thinks!

Reply:
Please have a closer look at the old logic on creating the return value. It will return true, when only one PersistanceProviders update succseeds. Thats a lot of unneeded code that can produce issues in the future. When implementing a new return value the use case should be specified before.